qa/test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# stop on error, and echo commands to stdout
+set -ex
+
+black --check src
```

Comment:
don't forget about pyflake, see https://github.com/tchx84/linka/blob/master/qa.sh

Reply:
Also, no need for the qa sub folder, just put it in the root dir as `qa.sh` is something I following in other of my projects as well :)

Reply:
Oh, and don't forget about the `qa-requirements.txt` file, instead of manually pip installing one by one on the CI side.

Reply:
Is it supposed to be called `requirements-test.txt` or `requirements-dev.txt` instead, as its the convention?

Reply:
It's neither of those, so just call it `qa-requirements.txt`

Reply:
ok